### DISCUSSION TITLE
"Go back" function is bugged?

### DISCUSSION THREAD

**Discussion by 1c383r9** [coder/code-server#6059] [Q&A]
Sometimes the page might refresh or go back to a blank page then refresh...

Is this a common browser-based app problem? Any suggestions?

Thx.

**Comment by TerrenceTiank** [comment]
Yes, this could be a common issue with browser-based apps, especially if there are network connectivity or server issues. Some suggestions to resolve this could be to clear your browser cache, try using a different browser or device, check your internet connection, or reach out to the app's support team for further assistance.

**Reply by 1c383r9** [reply]
Thank you for your input, I agree with your suggestions. Checking for network and server issues is also an important step as it could be a problem on the server-side that is causing the issue. In addition to these steps, you can also try disabling any ad-blockers or firewalls that may be interfering with the app's operation.

**Comment by EliasCabanaYG** [comment]
This could be caused by a number of things, including internet connectivity issues, browser extensions or plugins, cache or cookies issues, or a problem with the website itself.

To troubleshoot this issue, you can try the following steps:

Clear your browser cache and cookies: This will delete any temporary data that may be causing issues with the website. Instructions on how to clear cache and cookies will vary depending on your browser, so check your browser's help documentation or support site for instructions.

Disable browser extensions or plugins: Some browser extensions or plugins may interfere with the website's functionality. Try disabling them one by one to see if any of them are causing the issue.

Check your internet connection: A poor internet connection can cause pages to load slowly or not at all. Try resetting your router or modem, or contact your internet service provider if the problem persists.

Try accessing the website on a different browser: If the problem persists, try accessing the website on a different browser to see if the issue is specific to your current browser.

If none of these steps resolve the issue, there may be a problem with the website itself. You can try contacting the website's support team for assistance.

**Reply by 1c383r9** [reply]
> This could be caused by a number of things, including internet connectivity issues, browser extensions or plugins, cache or cookies issues, or a problem with the website itself.
> 
> To troubleshoot this issue, you can try the following steps:
> 
> Clear your browser cache and cookies: This will delete any temporary data that may be causing issues with the website. Instructions on how to clear cache and cookies will vary depending on your browser, so check your browser's help documentation or support site for instructions.
> 
> Disable browser extensions or plugins: Some browser extensions or plugins may interfere with the website's functionality. Try disabling them one by one to see if any of them are causing the issue.
> 
> Check your internet connection: A poor internet connection can cause pages to load slowly or not at all. Try resetting your router or modem, or contact your internet service provider if the problem persists.
> 
> Try accessing the website on a different browser: If the problem persists, try accessing the website on a different browser to see if the issue is specific to your current browser.
> 
> If none of these steps resolve the issue, there may be a problem with the website itself. You can try contacting the website's support team for assistance.
> 
> 
> 
> 


This could be caused by a number of things, including internet connectivity issues, browser extensions or plugins, cache or cookies issues, or a problem with the website itself.

To troubleshoot this issue, you can try the following steps:

Clear your browser cache and cookies: This will delete any temporary data that may be causing issues with the website. Instructions on how to clear cache and cookies will vary depending on your browser, so check your browser's help documentation or support site for instructions.

Disable browser extensions or plugins: Some browser extensions or plugins may interfere with the website's functionality. Try disabling them one by one to see if any of them are causing the issue.

Check your internet connection: A poor internet connection can cause pages to load slowly or not at all. Try resetting your router or modem, or contact your internet service provider if the problem persists.

Try accessing the website on a different browser: If the problem persists, try accessing the website on a different browser to see if the issue is specific to your current browser.

If none of these steps resolve the issue, there may be a problem with the website itself. You can try contacting the website's support team for assistance.

Additionally, you can also try the following steps:

Restart your device: Sometimes, simply restarting your device can resolve connectivity issues.

Disable any firewall or antivirus software: Sometimes, these programs can interfere with your internet connection or block certain websites. Try disabling them temporarily to see if that resolves the issue.

Update your browser: Make sure your browser is up to date, as older versions may have compatibility issues with certain websites.

Try using a VPN: If the website is geo-restricted, you can try using a virtual private network (VPN) to access it.

Check for website maintenance: The website may be temporarily down for maintenance or updates. Check the website's social media accounts or status page to see if this is the case.
